boefjes/boefjes/plugins/kat_dns_version/normalize.py

```diff
@@ -0,0 +1,108 @@
+import json
+from collections.abc import Iterable
+
+from boefjes.job_models import NormalizerOutput
+from octopoes.models import Reference
+from octopoes.models.ooi.findings import CVEFindingType, Finding
+from octopoes.models.ooi.software import Software, SoftwareInstance
+from packaging.version import Version
+
+cves = {
+    "bind": {
+        "CVE-2024-0760": "A flood of DNS messages over TCP may make the server unstable https://kb.isc.org/docs/cve-2024-0760",
+        "CVE-2024-1737": "BIND's database will be slow if a very large number of RRs exist at the same name https://kb.isc.org/docs/cve-2024-1737",
+        "CVE-2024-1975": "SIG(0) can be used to exhaust CPU resources https://kb.isc.org/docs/cve-2024-1975",
+        "CVE-2024-4076": "Assertion failure when serving both stale cache data and authoritative zone content https://kb.isc.org/docs/cve-2024-4076",
+    }
+}
+
+
+def run(input_ooi: dict, raw: bytes) -> Iterable[NormalizerOutput]:
+    input_ooi_reference = Reference.from_str(input_ooi["primary_key"])
+    input_ip = input_ooi["address"]["address"]
+    input_port = input_ooi["port"]
+
+    results = json.loads(raw)
+    for version in results:
+        name = None
+        versionnumber = None
```

Comment:
```suggestion
        version_number = None
```

Reply:
You can also initialize the `name` an `version_number` in an else-clause below. This way you have a more clear flow of conditions ensuring all branches are handled and it also prevents a "unused variable error"